macros/LearnSidebar.ejs

```diff
@@ -710,6 +710,31 @@ var text = mdn.localStringMap({
       'Tools_and_setup': '工具與設定',
       'Design_and_accessibility': '設計與親合度',
     'How_to_contribute': '如何貢獻'
+  },
+  'fr-FR': {
+    'Complete_beginners': 'Les débutants commence ici!',
+    'Getting_started_with_the_web': 'Commencer avec le Web',
+      'Getting_started_with_the_web_overview': 'Vue d\'ensemble de Commencer avec le Web',
+      'Installing_basic_software': 'Installation des outils de base',
+      'What_will_your_website_look_like': 'Quel sera l\'aspect de votre site web ?',
+      'Dealing_with_files': 'Gérer les fichiers',
+      'HMLT_basics': 'Les bases du HTML',
```

Comment:
Be careful of the slug HTML instead of HMLT. I would also suggest "Les bases de HTML" in order to be consistent with the following.

Reply:
Arf thanks, I'll change that!